README.md

```diff
@@ -1,2 +1,32 @@
 # Objective Revision Evaluation Service
 A webserver for hosting scoring services.
+
+# Installation
+
+## Vagrant
+The recommended installation is to set up Vagrant and VirtualBox, then `cd ores` and run:
+
+* `vagrant up`
```

Comment:
Should have step 1 and 2 as "install virtualbox" and "install vagrant" with links to their docs.